src/crimson/common/buffer_io.cc

```diff
@@ -0,0 +1,34 @@
+#include "buffer_io.h"
+
+#include <seastar/core/reactor.hh>
+#include <seastar/core/fstream.hh>
+#include <seastar/core/do_with.hh>
+
+#include "include/buffer.h"
+
+namespace ceph::buffer {
+
+seastar::future<> write_file(const ceph::buffer::list&& bl,
```

Comment:
What's the reason for `const &&`?

Reply:
i failed to remove `const` when changing the type of `bl` from `const ceph::buffer::list&` to `ceph::buffer::list&&`, when i realized that it'd be easier to pass-by-move the buffer to the continuation of underlying `seastar::file`.